requirements.txt

```diff
@@ -69,10 +43,6 @@ sniffio==1.3.0
     #   anyio
     #   httpcore
     #   httpx
-tomli==2.0.1
-    # via
-    #   coverage
-    #   pytest
 tree-sitter==0.20.1
     # via -r requirements.in
 urllib3==1.26.10
```

Comment:
You could remove this dependency (and others) from `setup.py` since it's a transitive dependency (which you know since it says `via {other_package}`